src/utils/__tests__/legacyKmlUtils.spec.js

```diff
@@ -79,7 +78,9 @@ function performStandardChecks(
     expectedFeatureType,
     expectedTitle = '',
     expectedDescription = '',
-    expectedCoordinateCount = 2
+    // TODO with the PR #X the ol geometries parsed from KML are with the layout 'xyz' instead
+    // of 'xy' therefore the expectedCoordinateCount has been increase to 3.
```

Comment:
is that meant to be amended or removed? It is missing a proper PR number (or if not available, we could default back to a JIRA ticket ID)

Reply:
I'll remove it, I think the changes of the coordinates from 'xy' to 'xyz' comes from the fact that we were overwriting the feature geometry coordinate from the one in the EditableFeature, while now we do the oposite.